brave/brave-core
# fix(vpn): Remove VPN promotion callout

Resolves https://github.com/brave/brave-browser/issues/42184



## Submitter Checklist:

- [x] There is a [ticket](https://github.com/brave/brave-browser/issues/42184
) for my issue
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan: (not a debug build)
1. Skip the onboarding flow
2. Dismiss the NTP intro messages
3. Kill the app
4. Set the device date ahead by 4 days if it's production build install
5. Relaunch app
6. observe Vpn callout is not happening
